client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/standalone/StandaloneAnalysisConfiguration.java

```diff
@@ -17,39 +17,31 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonarsource.sonarlint.core.client.api.analysis;
+package org.sonarsource.sonarlint.core.client.api.standalone;
 
 import java.nio.file.Path;
 import java.util.Collections;
 import java.util.LinkedHashMap;
 import java.util.Map;
-import javax.annotation.CheckForNull;
-import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
+import org.sonarsource.sonarlint.core.client.api.common.analysis.ClientInputFile;
 
 @Immutable
-public class AnalysisConfiguration {
+public class StandaloneAnalysisConfiguration {
 
-  private final String moduleKey;
   private final Iterable<ClientInputFile> inputFiles;
   private final Map<String, String> extraProperties;
   private final Path workDir;
   // TODO baseDir is only used to please Sensor API. To be removed when having a dedicated SonarLint API.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)